dashboard/menu/discovery/pages.md

```diff
@@ -1,19 +1,217 @@
+
+Unless you specify further inclusion rules and fine-tune your scope with exclusion rules, the proxy will ignore all pages outside of this directory.
+
+#### Except
+
+Exception rules don't need to be enabled "generally" in the way that inclusion rules do: if you add an URL prefix as an exclusion and leave the checkbox to its left enabled, it will *always* be applied.
+
+### Rule Application
+
+Whenever the proxy comes across a page in an affected context (crawling, serving, translating, etc.), it will evaluate it according to the rules you provide. This process is summarized in the flowchart below:
+
+![Evaluation of Inclusion/Exclusion Rules](/img/dot-graphs/inclusion-rules-eval.png)
+
+A few points of note concerning inclusion/exclusion rules:
+
+- pathnames are first checked for inclusion, then exclusion.
+
+- a pathname needs to match a single rules from the enabled set. Each such rule is applied to the path in sequence until a match is found or there are no more rules.
+
+- the rules are **strings** and matched from the beginning of the pathname. The proxy does not analyze them in detail or produce complex internal representations.
+
+- query parameters are supported (but be careful, you can't really count on a query parameter to have a set position!).
+
+- if a path falls outside of the scope of your inclusion rules or an exclusion rule applies to it, it will be greyed out in the page list and the text "Excluded by rule" will be visible next to it.
+
+- Rule-excluded paths cannot be un-excluded directly or via the Mass exclusion dialog, and the hover menu button you can ordinarly use for the purpose will be missing. You need to edit your rules if they gobble a path that you want.
+
+**NOTE:** If all pages are excluded on a project, crawls *cannot be started*, even if the currently active rules would allow for the inclusion of *some, as-of-yet undiscovered page*. In this case, crawls will exit after 0 pages visited. You have to ensure an entry point: that at least one of the known pages is in an included state. Otherwise, the crawler can't set its foot in the door.
+
+Though it may seem nonsensical to exclude all projects on a project, we note the case because such a state can come about from inadvertent use of *inclusion rules*.
+
+Consider, for example, that if you set `/en/` as the sole "Include only" rule on your project, but no page starting with `/en/` is in the page list, then not a single valid entry point is provided to the crawler.
+
+## Page List
+
+The page list contains all pages that the proxy has seen and collected previously, either via crawls or via user visits (to prevent the page list from being automatically extended in this latter fashion, enable "Page Freeze" in Advanced settings).
```

Comment:
@lsoos Are we able to provide a link to a description explaining Page Freeze more in details?